docs/content/usages/assertions.md

```diff
@@ -0,0 +1,250 @@
+---
+sort: 6
+---
+
+# Assertions
+
+Assertions are a set of action properties available to every action to assert the result after its execution.
+
+Assertions are executed after a handler and an error is thrown if the assertion doesn't validate. The name of the properties is prefixed with `assert` or `unassert` for their negation.
+
+## Example
+
+Assertion a directory was removed and newly created: 
+
+```js
+(async () => {
+  await nikita
+  // Asserts a directory is removed
+  .fs.remove({
+    // highlight-next-line
+    unassert_exists: '/tmp/a_dir',
+    recursive: true,
+    target: '/tmp/a_dir'
+  })
+  // Assert a directory is newly created
+  .fs.mkdir({
+    // highlight-range{1-2}
+    assert: {status: true},
+    assert_exists: '/tmp/a_dir',
+    target: "/tmp/a_dir"
+  })
+})()
+```
```

Comment:
For example, the `unassert_exists` ensures that a file or directory is not present once the `fs.remove` action is executed:
...
You can also combine two assertions. In such case, both assertions must succeed:
...